modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AddWriteCommittedInvokeClosure.java

```diff
@@ -53,7 +54,11 @@ class AddWriteCommittedInvokeClosure implements InvokeClosure<VersionChain> {
 
     private @Nullable VersionChain newRow;
 
-    private boolean addToGc;
+    /**
+     * Link to the row version for which the commit occurred. It will be a {@link PageIdUtils#NULL_LINK} if the current and the previous
+     * row versions are tombstones or have only one row version in the version chain.
```

Comment:
This comment is very confusing actually. Can you please elaborate on it?
Maybe a rename would help. It's a link that would be added to a GC queue after the closure execution, right? It's hard to decipher from the name, cause it looks more like "rowLink that will be processed by GC"

Reply:
Tried to fix it.